week3/helpMenu.txt

```diff
@@ -0,0 +1,63 @@
+
+    HELP MENU                                                                                    ||
+=================                                                                                ||
+                                                                                                 ||
+Instructions:                                                                                    ||
+                                                                                                 ||
+---------------------------------------                                                          ||
+$ curl -X GET 'http://localhost:8080/' >>>>>>>>>>> To display the help menu                      ||
+---------------------------------------                                                          ||
+                                                                                                 ||
+---------------------------------------                                                          ||
+$ curl -X GET 'http://localhost:8080/help' >>>>>>>>>>> To display the help menu                  ||
+---------------------------------------                                                          ||
+                                                                                                 ||
+-----------------------------------------------------                                            ||
+$ curl -X POST 'http://localhost:8080/add/{newTask}' >>>>>>>>>>> To add a task to the list,      ||
+-----------------------------------------------------            where {newTask} without spaces. ||
+                                                                                                 ||
+-----------------------------------------------------                                            ||
+$ curl -X DELETE 'http://localhost:8080/delete/{taskId}' >>>>>>>>>>> To remove a task from the   ||
```

Comment:
Now, this is actually okay, sort of. You can read up on REST style - REST is an overused term, but what it actually means is an organised way to structure the URL paths. In REST style, a deletion would actually be:

`curl -X DELETE 'http://localhost:8080/todo/{taskId}`

We don't need the path to include `delete`, because we have that information already from the HTTP verb.